tests/test_enterprise_learner_portal/api/test_views.py

```diff
@@ -238,15 +238,45 @@ def test_view_requires_openedx_installation(self):
                 )
             )
 
-    @mock.patch('enterprise_learner_portal.api.v1.views.EnterpriseCourseEnrollmentSerializer')
+    @mock.patch('enterprise_learner_portal.api.v1.views.get_resume_urls_for_course_enrollments')
     @mock.patch('enterprise_learner_portal.api.v1.views.get_course_overviews')
-    def test_patch_success(self, mock_get_overviews, mock_serializer):
+    @mock.patch('enterprise_learner_portal.api.v1.serializers.get_course_run_status')
+    @mock.patch('enterprise_learner_portal.api.v1.serializers.get_emails_enabled')
+    @mock.patch('enterprise_learner_portal.api.v1.serializers.get_course_run_url')
+    @mock.patch('enterprise_learner_portal.api.v1.serializers.get_certificate_for_user')
+    def test_patch_success(
+        self,
+        mock_get_cert,
+        mock_get_course_run_url,
+        mock_get_emails_enabled,
+        mock_get_course_run_status,
+        mock_get_overviews,
+        mock_get_resume_urls,
+    ):
         """
         View should update the enrollment's saved_for_later field and return serialized data from
          EnterpriseCourseEnrollmentSerializer for the enrollment (which we mock in this case)
         """
-        mock_get_overviews.return_value = {'overview_info': 'this would be a larger dict'}
-        mock_serializer.return_value = self.MockSerializer()
+        mock_get_overviews.return_value = [{
+            'id': self.course_run_id,
+            'start': 'a datetime object',
+            'end': 'a datetime object',
+            'display_name_with_default': 'a default name',
+            'pacing': 'instructor',
+            'display_org_with_default': 'my university',
+        }]
+        mock_get_resume_urls.return_value = {
+            self.course_run_id: '/courses/course-v1:MITx+6.86x+2T2024'
```

Comment:
nit: The API returns the `resume_course_run_url` as an absolute URL, versus the relative URL depicted here. Might be good to make the mocked URL absolute to have parity with the actual URL structure?

Reply:
The serializer will actually build an absolute url out of whatever [get_resume_urls_for_course_enrollments()](https://github.com/openedx/edx-platform/blob/7cf97ec0de812659f69f565eff24f7fa342d1d49/lms/djangoapps/learner_home/views.py#L261C5-L261C43) returns. And since the above function uses `reverse()` to get the course resume url, a relative path is what we'll get before the serializer builds an absolute URL - so the mock is already set up in a way that matches what we'd see in a real request.